### ISSUE TITLE
Unlocking with fingerprint

### ISSUE THREAD

**Issue by mihaelmilea** [ubports/ubuntu-touch#224]
- Device: Turbo
- Channel: Any but currently devel
- Build: Any but currently ing20

## Steps to reproduce

1. In order to unlock the phone I first press (deep hardware press) the fingerprint button. This way the phone screen comes on and it waits for the unlock code or the fingerprint.
2. Then I lift my finger from the button and then I touch the button again - this time I only touch the button, I did not press it.
3. The phone unlocks by reading the fingerprint

## Expected behavoir

I wish the phone would unlock without step number 1, which means just by a short touch of the fingerprint reader.

## Actual behavior

As I described it it is quite a work to do to unlock the phone - something that needs to be done every time I use the phone.

**Comment by NeoTheThird** [comment]
Thanks for reporting. I see how that would be nice, but i'm uncertain if it's possible without preventing the phone from going to sleep. That will have to be investigated.

What exactly do you mean by this?
> Channel: Any but currently devel
> Build: Any but currently ing20

I would assume it affects all devices and channels?

**Comment by mihaelmilea** [comment]
Yes, I mean all channels and builds. Sorry for the confusion. There is a bug when I type: It ads the last word I typed whenever I click somewhere else and I press space.

**Comment by NeoTheThird** [comment]
Ahh, yeah. That's a known issue...

**Comment by Marathon2422** [comment]
Fingerprint,not working at all (on latest ,version 4,now), I get no response at all from the fingerprint reader,( I did change the fingerprint setup,no difference).

**Comment by mihaelmilea** [comment]
Mine works...

**Comment by NeoTheThird** [comment]
@Marathon2422 What Channel are you on? When did that behavior first appear?

**Comment by Marathon2422** [comment]
I am on 15.04. (r4),noticed this on (r3)~ I was hoping / expecting it to be 
fixed.
  I will most likely do a wipe and reinstall,just to be sure its not just 
me.


-- 
Sent using Dekko from my Ubuntu device

**Comment by Marathon2422** [comment]
I will most likely,wipe and reinstall later,to be sure its not just me 
(works on Flyme)


-- 
Sent using Dekko from my Ubuntu device

**Comment by Marathon2422** [comment]
fyi
I still have the same problem,I did use twrp ,wiped,played with multirom,then reinstalled,stable,then 15.04rc.

**Comment by NeoTheThird** [comment]
@Marathon2422 You "Played with Multirom"? That means you're running Ubuntu Touch with Multirom? Then that's probably the issue.

**Comment by Marathon2422** [comment]
No ,completely  wiped trying devel,version 32 now .

**Comment by NeoTheThird** [comment]
If @mihaelmilea can't reproduce that issue, it might be a hardware problem.

**Comment by Marathon2422** [comment]
It works now, after I installed devel r32, I made a new fingerprint, and set the fingerprint as 1st choice to unlock and yahoo it works,most likely my equipment,testing me.
( it did work in android,and Flyme)

**Comment by UniversalSuperBox** [comment]
That's great to hear! Thanks for keeping with us.

**Comment by mihaelmilea** [comment]
Wasn't this hijacked and closed when in fact I opened it about something else? ... :)

**Comment by UniversalSuperBox** [comment]
Indeed. I'll reopen it.
